walkthroughs/week-4-libraries/maps/examples/info-window/src/main/webapp/index.html

```diff
@@ -2,7 +2,7 @@
 <html>
 <head>
   <title>Google Maps Info Window Example</title>
-  <script src="https://maps.googleapis.com/maps/api/js?key=YOUR_API_KEY"></script>
+  <script src="https://maps.googleapis.com/maps/api/js?key=AIzaSyCi6TvSYqvVOLcQkPoWgdWQFgyqVTxJYsg"></script>
```

Comment:
API Keys should never be committed, especially to a public repository. "Rolling forward" by deleting it in the next PR won't delete this from your Git history. It's exposed now, since this is already pushed to GitHub.

See: https://docs.google.com/document/d/12pE_7GRG3efHyZsUgWueoSGi7xM4t-7FoojYEFNX0I0/edit#bookmark=id.kqkn8rvu9dsm

You should destroy this API key now.

Reply:
destroyed.

Reply:
As discussed in Chat last week, turns out I was overzealous here. Since the Maps API key is used by the JS on the client side, it's actually necessary to expose it. It's fine to expose it, but you should apply these restrictions:
https://developers.google.com/maps/documentation/javascript/get-api-key#restrict_key